### ISSUE TITLE
cli `fastly:conf:set --activate` doesnt activate anything

### ISSUE THREAD

**Issue by jbartus** [fastly/fastly-magento2#653]
it doesnt return an error or activate anything alone, it appears that it has to be used in conjunction with `--upload-vcl` to work

```
# ./bin/magento fastly:conf:set --upload-vcl         <--- works alone
Successfully uploaded VCL. 
Flushed cache types:
config

# ./bin/magento fastly:conf:set --activate       <--- seems to do nothing
Flushed cache types:
config

# ./bin/magento fastly:conf:set --upload-vcl --activate   <---   work together
Successfully uploaded VCL. Activated Version 5
Flushed cache types:
config
```

an error check that returns something like "must be used with --upload-vcl" would be enough to have caught me.

**Comment by dpotkoc** [comment]
Hello @jbartus 

**php bin/magento fastly:conf:set --help** 
command has explanation: "**Activate newly cloned version. Used with VCL upload.**"

If you didn't make any modifications then there is no sense to call this command, there is no new version to activate.

**Comment by vvuksan** [comment]
The issue is that -activate by itself doesn't do anything. So we need a
check that catches that.

**Comment by vvuksan** [comment]
Will be released as 1.2.205
